theme/snippets/add-to-cart-container.liquid

```diff
@@ -0,0 +1,34 @@
+<!-- Assign label and disabled state to add to cart button -->
+{% capture button_label %}{{"snippets.add_to_cart_container.add_to_cart_btn.text" | t }}{% endcapture %}
+{% assign button_disabled = false %}
+{% if product.available %}
+  {% if product_in_cart %}
```

Comment:
Noticed one thing, we can remove this case, where we disable the button and update the label to "added to cart" if the item is currently in the cart. But feel free to do that in the follow up where you add the variant dropdown

Reply:
TY!! Will add to Pt 2 #174 :)